packages/pwa-kit-dev/src/utils/resolver-utils.test.js

```diff
@@ -147,6 +147,7 @@ describe('resolverUtils', () => {
                 extensions: ['module-extension-a', 'module-extension-b', 'module-extension-c'],
                 expected: [
                     path.join(process.cwd(), 'app', 'overrides', 'app', 'routes'),
+                    path.join(process.cwd(), 'app', 'app', 'routes'),
```

Comment:
Can we double check that this path is correct? It contains duplicate 'app'. Should it be more like this?
```suggestion
                    path.join(process.cwd(), 'app', 'routes'),
```

Reply:
Maybe the fix can be simply having the import path to be:
initially `*/app/routes` → now `*/routes`